Package.swift

```diff
@@ -22,7 +22,7 @@ func resolveTargets() -> [Target] {
         .target(name: "Purchases",
                 dependencies: [],
                 path: ".",
-                sources: ["Purchases", "Purchases/Public", "Purchases/Caching"],
+                sources: ["Purchases", "Purchases/Public"],
```

Comment:
I think this is a wrong conflict resolution

Reply:
I'm honestly not sure what to do here. 
in theory, sources is supposed to recursively add subfolders, so you should only really need `["Purchases"]`. 
However, if I remove "Public", I get a compilation error, undefined symbol. 
For the `Caching` folder, though, it works fine if you don't add it here. 
Ideally I'd like to figure out why the reference to Public is needed, and if possible remove it.